content/pages/index.html

```diff
@@ -118,8 +118,6 @@ <h3 id="heading">Create IEEE 829 or free text Test Plan</h3>
             </div>
             <div class="col-2 screenshot_switch_arrow accent-important-icon" id="next">
                 <div class="arrow-right"><i class="fa fa-arrow-right fa-5x"></i></div>
-            </div>
-        </section>
```

Comment:
In both index.html and features.html you should add comments at the beginning of the content and at the end of the content noting that the opening and closing tags are inside page.html. Otherwise the content appears to be missing some HTML tags (and it is). This is to prevent confusion and broken HTML in the future.


Note about the script tags: with the new changes they really should move out of the index.html file because they do look quite ugly. However it is OK to do this later or in a separate commit.